src/EditorFeatures/Test2/GoToDefinition/GoToDefinitionCommandHandlerTests.vb

```diff
@@ -66,13 +65,14 @@ class C
                 Dim mockDocumentNavigationService =
                     DirectCast(workspace.Services.GetService(Of IDocumentNavigationService)(), MockDocumentNavigationService)
 
-                GoToDefinitionCommandHandler.TryExecuteCommand(view.TextSnapshot, baseDocument.CursorPosition.Value, TestCommandExecutionContext.Create())
+                Dim handler = New GoToDefinitionCommandHandler()
```

Comment:
📝 This should be obtained through the MEF catalog instead of directly constructed.